### PR TITLE
Create histories_transfer_entire_histories.md

### PR DESCRIPTION
**Pull request by AnomalyCodes**
This FAQ contribution addresses the issue of history transfer from one galaxy instance to another.

### CODE REVIEW

File: faqs/galaxy/histories_transfer_entire_histories.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+---
+To transfer an entire history from one galaxy instance to another, follow these steps:
+
+1. Click on {% icon galaxy-gear %} in the history panel of the *sender* galaxy instance
+2. Click on **Export to File**
+3. Select either exporting history **to a link** or **to a remote file**
+4. Click on the link text to generate a new archive for the history *if* exporting to a link
+5. Wait for the link to generate
+6. Copy the link address or click on the generated link to download the history archive
+7. Click on **User** on the top menu of the *receiver* galaxy instance
```
Comment (beatrizserrano) [inline faqs/galaxy/histories_transfer_entire_histories.md:16]:
```suggestion
7. Click on **User** on the top menu of the *receiver* Galaxy instance
```

**Review by beatrizserrano — COMMENTED** [review]
Thanks a lot @AnomalyCodes! I just made a couple of minor comments, the rest looks quite good to me, you did a great job creating clear steps and summarising :)

File: faqs/galaxy/histories_transfer_entire_histories.md (hunk 1 above)
Comment (AnomalyCodes) [reply faqs/galaxy/histories_transfer_entire_histories.md:16]: Thank you, @beatrizserrano for this suggestion!

File: faqs/galaxy/histories_transfer_entire_histories.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Transfer entire histories
+area: histories 
+box_type: tip
+layout: faq
+contributors: [AnomalyCodes]
+---
+To transfer an entire history from one galaxy instance to another, follow these steps:
```
[the remark this answers is not in the document]
Comment (AnomalyCodes) [reply faqs/galaxy/histories_transfer_entire_histories.md:8]: I will update the title now. Thank you!

**Comment by AnomalyCodes** [comment]
> Thanks a lot @AnomalyCodes! I just made a couple of minor comments, the rest looks quite good to me, you did a great job creating clear steps and summarising :)

Thank you @beatrizserrano! I will apply your suggestions now

**Review by beatrizserrano — DISMISSED** [review]

**Comment by beatrizserrano** [comment]
Oh @AnomalyCodes, forgot that you need to add yourself to the contributors before we can merge :)

File: faqs/galaxy/histories_transfer_entire_histories_from_one_galaxy_instance_to_another.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Transfer entire histories from one Galaxy instance to another
```
Comment (shiltemann) [inline faqs/galaxy/histories_transfer_entire_histories_from_one_galaxy_instance_to_another.md:2]: I think the word "instance" is a bit on the technical side for most users, you can just leave out the word? or use 'server' instead

```suggestion
title: Transfer entire histories from one Galaxy to another
```

**Comment by AnomalyCodes** [comment]
> Oh @AnomalyCodes, forgot that you need to add yourself to the contributors before we can merge :)

I have done that, @beatrizserrano!

**Pull request merged**